src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -410,10 +403,37 @@ public void actionPerformed(final ActionEvent e) {
     });
     menuGame.add(mapZoom).setMnemonic(KeyEvent.VK_Z);
   }
+  
+  private void addUnitNationDrawMenu(final JMenu parentMenu){
+    final JMenu unitSizeMenu = new JMenu();
+    unitSizeMenu.setMnemonic(KeyEvent.VK_N);
+    unitSizeMenu.setText("Show Unit Nation");
+    final ButtonGroup unitNationGroup = new ButtonGroup();
+    final JRadioButtonMenuItem nationOption0 = new JRadioButtonMenuItem("Hide Nation");
+    final JRadioButtonMenuItem nationOption1 = new JRadioButtonMenuItem("Next to Unit");
+    final JRadioButtonMenuItem nationOption2 = new JRadioButtonMenuItem("Below Unit");
```

Comment:
DRY violation, 5 lines of code where only two things change. I suggest a function call similar to:
`createRadioButtonItem("Below Unit", 0);`